app/views/homepage/_more_on_govuk.html.erb

```diff
@@ -1,15 +1,17 @@
-<section class="homepage-more-on-govuk">
-  <%= render "govuk_publishing_components/components/heading", {
-    font_size: "m",
-    margin_bottom: 6,
-    text: t("homepage.index.more"),
-  } %>
-  <%= render "govuk_publishing_components/components/heading", {
-    text: t("homepage.index.most_active"),
-    margin_bottom: 4,
-    font_size: "s",
-    heading_level: 3,
-  } %>
+<section class="homepage-section homepage-more-on-govuk">
```

Comment:
I think `homepage-section` as a means of providing consistent spacing is a good idea. It's out of scope of this PR but could you apply it to other sections on this page whilst we're in the same neighbourhood?

Reply:
I'd also love it if we got rid of the `govuk-width-container` div around the government activity, featured and more on govuk sections but only if it's not a pain in the backside.

Reply:
This was a slightly tricky one to get right but I've got something for you to have a look at now.

The main problem I had is the variation in sections one (`padding: 30px 0 50px`) and two (`padding: 30px 0 30px`) against all the sections (`padding: 20px 0 30px`).

However I think i've managed to organise things reasonably well, reusing the `homepage-section` pattern and with the necessary varations.